core/src/main/java/com/sequenceiq/cloudbreak/service/image/PlatformStringTransformer.java

```diff
@@ -39,7 +39,7 @@ public ImageCatalogPlatform getPlatformStringForImageCatalog(ImageCatalogPlatfor
     }
 
     public ImageCatalogPlatform getPlatformStringForImageCatalogByRegion(String cloudPlatform, String region) {
-        if (region.toLowerCase().contains("-gov-")) {
+        if (StringUtils.isNotBlank(region) && region.toLowerCase().contains("-gov-")) {
```

Comment:
Strings.isNullOrEmpty used in this class. Should we use the method here as well just to be general ?

Reply:
right, generalized